lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -1218,6 +1218,10 @@ void MtohRenderOverride::_InitHydraResources(const MHWRender::MDrawContext& draw
     //Add the scene index as an input scene index of the merging scene index
     _renderIndexProxy->InsertSceneIndex(_mayaHydraSceneIndex, SdfPath::AbsoluteRootPath());
     
+    if (!_sceneIndexRegistry) {
```

Comment:
This change is not really related to the bug but I think it's better to pull out this from the _CreateSceneIndicesChain function as it will be ignored most of the time.